src/_locales/en/messages.json

```diff
@@ -244,5 +244,9 @@
    "inPageUI-tooltip-button-share-passive": {
      "message": "If you click this button, Facebook will be able to track your visit to this site.",
      "description": ""
+   },
+   "fbcSettingsTitle": {
+     "message": "Preferences",
```

Comment:
Firefox is moving to "Settings", is there a reason to use "Preferences"?

Reply:
New title is "Optional Settings"

Reply:
I've never seen the concept of "Pptional settings" outside of technical configuration files. What's the reasoning behind this choice, vs just "Settings"?

Reply:
All of the settings are opt-in. I'm overthinking it. I'll switch it to "Settings" 

One note: when on `about:addons`, and digging into the settings panel for a specific add-on, they use "Preferences" 

<img width="716" alt="image" src="https://user-images.githubusercontent.com/2692333/168142786-5d79d058-9579-4ebe-8a3c-6c8bf1f3c32f.png">

Reply:
> One note: when on `about:addons`, and digging into the settings panel for a specific add-on, they use "Preferences"
> 
> <img alt="image" width="716" src="https://user-images.githubusercontent.com/2692333/168142786-5d79d058-9579-4ebe-8a3c-6c8bf1f3c32f.png">

That's probably worth a bug. I don't think the use of Settings was enforced, besides Firefox main Settings (previously Preferences/Options), but it would be nice to have consistency.